probe/probe.go

```diff
@@ -625,7 +626,7 @@ func getPoliciesOnAnnotatedPods(c *k8s.Client) (map[string][]string, error) {
 			}
 		}
 	} else {
-		fmt.Printf("No Resource found in namespace\n")
+		fmt.Printf("No kubearmor policies found\n")
 	}
```

Comment:
Let's not keep this. Like empty policy section in the below section anyway means that there no policies right?

Reply:
right!